godtools/Managers/FirstLaunchInitializer.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  FirstLaunchInitializer.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 5/19/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class FirstLaunchInitializer: GTDataManager {
+    
+    private let magicId = "-358"
```

Comment:
It's random. I just picked something negative as I knew the rails API would never generate that. I happen to like the number 358.